src/index/repositories.props

```diff
@@ -50,6 +50,15 @@
         $(VsDevCmd) &amp; build -hostType Core -skipTests -bootstrapOnly
       </PrepareCommand>
     </Repository>
+    <Repository Include="machinelearning">
+      <Url>https://github.com/dotnet/machinelearning</Url>
+      <Projects>
+        src/Microsoft.*/**/*.csproj;
+      </Projects>
+      <PrepareCommand>
+        $(VsDevCmd) &amp; build
```

Comment:
Is this the "best" command to run? Some of the other commands explicitly specify other options (such as `SkipTests`)

Reply:
Need Skiptests, surely we don't want to run tests during indexing...

Reply:
You don't need to `$(VsDevCmd)`. You can just run `.\build.cmd` from the root of the machinelearning repo in any command prompt/powershell.

You don't need to `SkipTests`, because running tests is not enabled by default. Building just builds.  Instead we have:

`[-runtests]       Runs the tests.  Call this after building.`

If you want to run tests.

So in short, I would just drop `$(VsDevCmd)`.

Reply:
Side question: why do you need to build?

Reply:
I believe it caches the binaries, but I don't remember all the details of Kirill's code (https://github.com/KirillOsenkov/SourceBrowser).

I was going to keep $(VsDevCmd), even if we don't need it, because all the other entries have it as well (and I don't believe we need it there either).
* Perhaps there is some general requirement of the tooling?

Reply:
You don't actually need to build, the only thing that is required is restore and anything else that generates compile inputs. The indexer needs all the inputs to compile to be avaliable. The best way to do that is actually compile things, but if the repo is not complicated then restore should be fine.